cogs/deadline.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, bot):
                       help="put in current time to get offset needed for proper "
                            "datetime notifications $timenow MMM DD YYYY HH:MM ex. $timenow SEP 25 2024 17:02")
     async def timenow(self, ctx, *, date: str):
+        if not ctx.channel.name == 'assignments':
+            await ctx.author.send('Please send questions to the #assignments channel.')
```

Comment:
'Please send questions to the #assignments channel.'

Should be:

'Please use this command inside the #assignments channel.'